<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pydonia</class>
 <widget class="QMainWindow" name="Pydonia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>553</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Pydonia</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="MainFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="MainFrameLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="ToolsFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="ToolsFrameLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeView" name="FileMap">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDockWidget" name="SymbolMenu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="windowTitle">
             <string>Symbol Menu</string>
            </property>
            <widget class="QWidget" name="dockWidgetContents_3">
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>20</y>
                <width>49</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>20</y>
                <width>49</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>350</x>
                <y>20</y>
                <width>49</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
             <widget class="QProgressBar" name="progressBar">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>100</y>
                <width>118</width>
                <height>23</height>
               </rect>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
             <widget class="QLCDNumber" name="lcdNumber">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>80</y>
                <width>64</width>
                <height>23</height>
               </rect>
              </property>
              <property name="intValue" stdset="0">
               <number>42</number>
              </property>
             </widget>
             <widget class="QSlider" name="verticalSlider">
              <property name="geometry">
               <rect>
                <x>290</x>
                <y>10</y>
                <width>18</width>
                <height>160</height>
               </rect>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_5">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>40</y>
                <width>79</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_6">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>120</y>
                <width>79</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_7">
              <property name="geometry">
               <rect>
                <x>330</x>
                <y>140</y>
                <width>79</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="FileViewer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::TabShape::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <property name="tabBarAutoHide">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="WelcomeTab">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <attribute name="title">
           <string>Welcome</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Getting started:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Open or create a Project&lt;br/&gt;View the &lt;/span&gt;&lt;a href=&quot;https://duckduckgo.com/&quot;&gt;&lt;span style=&quot; font-weight:400; text-decoration: underline; color:#99ebff;&quot;&gt;documentation&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTerminal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
